industrial_ci/src/builders/colcon.sh

```diff
@@ -40,7 +40,13 @@ function builder_run_tests {
     else
         output_handler="console_cohesion+"
     fi
-    ici_exec_in_workspace "$extend" "$ws" colcon test --event-handlers "${_colcon_event_handlers[@]}" "${output_handler}"
+    local -a opts
+    if [ "$PARALLEL_TESTS" == false ]; then
+        opts+=("--executor sequential --parallel-workers 1")
+    else
+        opts+=("--executor parallel")
+    fi
+    ici_exec_in_workspace "$extend" "$ws" colcon test --event-handlers "${_colcon_event_handlers[@]}" "${output_handler}" "${opts}"
```

Comment:
```suggestion
    ici_exec_in_workspace "$extend" "$ws" colcon test  "${opts[@}"
```
"${var[@}" expands the array into individual strings.